### ISSUE TITLE
Bottlerocket AMIs only available in us-west-2

### ISSUE THREAD

**Issue by errordeveloper** [aws/containers-roadmap#827]
I have tried [eksctl example](https://github.com/weaveworks/eksctl/blob/89ba3db3f3a227503dc5b8fd00d36253953ce73b/examples/20-bottlerocket.yaml) for Bottlerocket nodegroup, but it didn't work in eu-west-1 region.

The error I got was this:
```
$ eksctl create cluster -f cluster.yaml 
[ℹ]  eksctl version 0.15.0
[ℹ]  using region eu-west-1
[ℹ]  setting availability zones to [eu-west-1a eu-west-1c eu-west-1b]
[ℹ]  subnets for eu-west-1a - public:192.168.0.0/19 private:192.168.96.0/19
[ℹ]  subnets for eu-west-1c - public:192.168.32.0/19 private:192.168.128.0/19
[ℹ]  subnets for eu-west-1b - public:192.168.64.0/19 private:192.168.160.0/19
Error: unable to determine AMI to use: error getting AMI from SSM Parameter Store: AccessDeniedException: No access to "/aws/" namespace: aws/service/bottlerocket is not a valid namespace
	status code: 400, request id: f8f44f27-7c44-4d7a-a5f5-ae43d9854840
$
```

To be honest I was confused, I thought I actually need to have preview access  to try the Bottlerocket AMI, but turned out it's a regional issues.

/cc @mhausenblas @samuelkarp

**Comment by sayboras** [comment]
Sad that it's only available in 4 regions ap-northeast-1, eu-central-1, us-east-1, us-west-2 as per https://github.com/bottlerocket-os/bottlerocket/blob/develop/QUICKSTART.md. 

I am waiting to try it out in ap-southeast-2 as well.

**Comment by samuelkarp** [comment]
Hello @errordeveloper!  As @sayboras pointed out, Bottlerocket is not yet available in eu-west-1.  You're welcome to use the images we've made available in one of the regions that we have already launched (ap-northeast-1, eu-central-1, us-east-1, and us-west-2).

If you want to use eu-west-1 before we've made images available, you can follow [these instructions](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md) to build and register your own AMI.  This will give you an image similar to our official ones (if you check out [this release](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v0.3.1), it should be the same).

**Comment by jhaynes** [comment]
Alternatively, you're welcome to copy our AMIs into new regions in your account. However, the way the host container ECR repository paths are generated by default includes a dynamic region component. You'll need to override those defaults with either ECR repos of your own or with one of our repos from one of our supported regions. 

The settings names are [here](https://github.com/bottlerocket-os/bottlerocket#host-containers-settings) and more details on how the defaults are set up is [here](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/models/defaults.toml#L53-L76)

**Comment by errordeveloper** [comment]
Perhaps someone should update the eksctl example to say which regions people can use, my starting point was the eksctl example and not the bottlerocket docs, I am sure others start there too :)

**Comment by runningman84** [comment]
It seems like there is already some automation in place to publish the AMI in 4 regions. I don't get why AWS can't just add few additional regions?

**Comment by errordeveloper** [comment]
That's kind of my point also...

**Comment by tjkirch** [comment]
In addition to the ap-northeast-1, ap-south-1, eu-central-1, us-east-1, and us-west-2 regions where Bottlerocket was already available, with today's 0.5.0 release we're also available in ap-northeast-2, ap-southeast-1, ap-southeast-2, ca-central-1, eu-north-1, **eu-west-1**, eu-west-2, eu-west-3, sa-east-1, us-east-2, and us-west-1.
